pt_os_web_portal/backend/routes.py

```diff
@@ -385,10 +384,17 @@ def post_disable_tour():
     return "OK"
 
 
-@app.route("/close-pt-browser", methods=["POST"])
-def post_close_pt_browser():
-    PTLogger.debug("Route '/close-pt-browser'")
-    close_pt_browser()
+@app.route("/close-os-updater-window", methods=["POST"])
+def post_close_os_updater_window():
+    PTLogger.debug("Route '/close-os-updater-window'")
+    close_window_by_title("pi-topOS Tour")
+    return "OK"
+
+
+@app.route("/close-pt-os-tour-window", methods=["POST"])
+def post_close_pt_os_tour_window():
+    PTLogger.debug("Route '/close-pt-os-tour-window'")
+    close_window_by_title("pi-topOS Updater Tool")
```

Comment:
I don't love that this is hard-coded like this.

Perhaps we should create entrypoints for each `web-renderer` process. e.g.: `pt-os-updater-ui` runs `web-renderer --window-title "pi-topOS Updater Tool" --hide-frame --size 0.7x0.7 --icon /usr/share/icons/Papirus/24x24/apps/system-software-update.svg`. By moving the generation of this command into the web portal's backend, the title can be fetched rather than duplicated here.

Reply:
This can be accepted as tech debt though